rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java

```diff
@@ -70,14 +70,14 @@
      *
      * @param sid session identifier
      */
-    void setSession(String sid);
+    void setSession(String sid) throws NotConnectedException;
```

Comment:
I'm afraid this change breaks one usage of the schedulerapi, which is to explicitely set the sessionid in order to connect to the scheduler.

it was indeed possible to do:
schedulerclient.setSession(sid)
schedulerclient.init(connectioninfo) // it will use the provided sessionid instead of the credentials
// use the api

Reply:
Did it as discussed. thanks!

Reply:
I rebased with master that is why it is outdated.